assets/product-form.js

```diff
@@ -12,7 +12,7 @@ if (!customElements.get('product-form')) {
     onSubmitHandler(evt) {
```

Comment:
Could be handled in a separate issue/PR - In Safari (iOS and macOS) the spinner inside the button when adding-to-cart is not visible.

I believe setting an explicit width to the svg fixes this, but requires more testing in all instances where it's being used.
```css
.loading-overlay__spinner svg {
  width: 100%;
}
```

![Kapture 2021-10-07 at 15 16 10](https://user-images.githubusercontent.com/3710548/136448411-11e4b0e5-8f7f-4503-b5a5-0309fc4a2010.gif)
e

Reply:
I'll fix this in a separate issue. Thanks for sharing a potential fix :D